src/test/ui/resolve/issue-46209-private-enum-variant-reexport.stderr

```diff
@@ -0,0 +1,26 @@
+error: enum is private and its variants cannot be reexported
+  --> $DIR/issue-46209-private-enum-variant-reexport.rs:12:13
+   |
+12 |     pub use self::Professor::*;
+   |             ^^^^^^^^^^^^^^^^^^^
+...
+18 |     enum Professor {
+   |     - help: consider making the enum public: `pub `
+
+error: variant is private and cannot be reexported
+  --> $DIR/issue-46209-private-enum-variant-reexport.rs:14:45
+   |
+14 |     pub use self::Lieutenant::{JuniorGrade, Full};
+   |                                             ^^^^
+...
+25 |     enum Lieutenant {
+   |     - help: consider making the enum public: `pub `
+
+error: variant is private and cannot be reexported
+  --> $DIR/issue-46209-private-enum-variant-reexport.rs:14:32
+   |
+14 |     pub use self::Lieutenant::{JuniorGrade, Full};
+   |                                ^^^^^^^^^^^
```

Comment:
Ideally I would like these to follow the same output as #43447, but it could get hairy fairly quickly...

Reply:
motion to defer